package.json

```diff
@@ -9,10 +9,13 @@
     "test": "echo \"Warning: no test specified\" && exit 0"
   },
   "dependencies": {
+    "@prodigy/game-framework": "7.0.0",
```

Comment:
Is this dependency going to get built into the web worker code and bloat it? Or will it all get tree-shaken out except for Vector2?

Reply:
We could just use a type compatible with framework's Vector2 to avoid this.